File: Belajar/Input_Output/Python/struk_barang.py
nama = input("Masukkan nama barang: ")
harga = float(input("Masukkan harga barang: "))
jumlah = int(input("Masukkan jumlah barang: "))

total = harga * jumlah

minimal_diskon = 40800
diskon_persen = 8

if total > minimal_diskon:
    diskon = total * (diskon_persen / 100)
else:
    diskon = 0

total_bayar = total - diskon

print("\n==== Struk Pembelian ====")
print("Nama Barang   :", nama)
print("Harga Satuan  : Rp", int(harga))
print("Jumlah        :", jumlah)
print("Total         : Rp", int(total))
print("Minimal Diskon: Rp", minimal_diskon)
print("Diskon (%)    :", str(diskon_persen) + "%")
print("Diskon        : Rp", int(diskon))
print("Total Bayar   : Rp", int(total_bayar))
print("=========================")

File: Belajar/Input_Output/Java/struk_barang.java
package Input_Output.Java;

import java.util.Scanner;

public class struk_barang {
    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);
        System.out.print("Masukkan nama barang: ");
        String nama = input.nextLine();
        System.out.print("Masukkan harga barang: ");
        double harga = input.nextDouble();
        System.out.print("Masukkan jumlah barang: ");
        
        int jumlah = input.nextInt();
        double total = harga * jumlah;
        int minimalDiskon = 40800;
        int diskonPersen = 8;
        double diskon;
        if (total > minimalDiskon) {
            diskon = total * (diskonPersen / 100.0);
        } else {
            diskon = 0;
        }

        double totalBayar = total - diskon;

        System.out.println("\n==== Struk Pembelian ====");
        System.out.println("Nama Barang   : " + nama);
        System.out.println("Harga Satuan  : Rp " + (int)harga);
        System.out.println("Jumlah        : " + jumlah);
        System.out.println("Total         : Rp " + (int)total);
        System.out.println("Minimal Diskon: Rp " + minimalDiskon);
        System.out.println("Diskon (%)    : " + diskonPersen + "%");
        System.out.println("Diskon        : Rp " + (int)diskon);
        System.out.println("Total Bayar   : Rp " + (int)totalBayar);
        System.out.println("=========================");
    }
}
